src/Microsoft.DotNet.ProjectModel/project.json

```diff
@@ -5,9 +5,8 @@
   },
   "description": "Types to model a .NET Project",
   "dependencies": {
-    "Microsoft.Extensions.DependencyModel": {
-      "target": "project"
-    },
+    "Microsoft.Extensions.DependencyModel": "1.0.1-beta-000914",
+    "Microsoft.DotNet.PlatformAbstractions": "1.0.1-beta-000914",
```

Comment:
If I had it here just for the transitive dependency, I would agree, but ProjectModel is actually using classes from PlatformAbstractions, so I would like to be explicit about the dependency.